mern/server/interactor/QA_interactor.js

```diff
@@ -0,0 +1,37 @@
+import QA_DAO from "../DAO/QA_dao.js";
+//Interactor calling the QA DAO functions after properly checking the inputs
+export default class QA_Interactor {
+  //Empty constructor
+  constructor(props) {
+    super(props);
+  }
+
+  //Gets the QA by ID
+  getQA(req, res) {
+    QA_DAO.getQAByID(req, res);
+  }
+
+  //Creates a new QA
+  createQA(req, res) {
+    QA_DAO.createQA(req, res);
+  }
+
+  //Updates QA if new QA object is valid and QA exists in collection
+  updateQA(req, res) {
+    const qa = QA_DAO.QAExists(req, res);
+    if (!qa) {
+      res.status(400, "QA not Found");
+    }
+    if (req.body.question && req.body.intents && req.body.question_included) {
```

Comment:
Can't simply do req.body.question. In testing, there were certain invalid values that passed this check.

Need to change to req.body.hasOwnProperty("question")